pkg/virt-controller/watch/vm_test.go

```diff
@@ -6155,6 +6155,52 @@ var _ = Describe("VirtualMachine", func() {
 					Entry("with the replacement updateVolumeStrategy",
 						kvpointer.P(v1.UpdateVolumesStrategyReplacement)),
 				)
+
+				It("should set the restart condition with the Migration updateVolumeStrategy if volumes cannot be migrated", func() {
+					testutils.UpdateFakeKubeVirtClusterConfig(kvInformer, &v1.KubeVirt{
+						Spec: v1.KubeVirtSpec{
+							Configuration: v1.KubeVirtConfiguration{
+								VMRolloutStrategy: &liveUpdate,
+								DeveloperConfiguration: &v1.DeveloperConfiguration{
+									FeatureGates: []string{
+										virtconfig.VMLiveUpdateFeaturesGate,
+										virtconfig.VolumesUpdateStrategy,
+										virtconfig.VolumeMigration,
+									},
+								},
+							},
+						},
+					})
+					vm, vmi := DefaultVirtualMachine(true)
+					vm.Spec.Template.Spec.Volumes = append(vm.Spec.Template.Spec.Volumes, v1.Volume{
+						Name: "vol1"})
+					vm.Spec.Template.Spec.Volumes = append(vm.Spec.Template.Spec.Volumes, v1.Volume{
```

Comment:
nit: you can merge the 2 volumes into a single append

Reply:
:man_facepalming: